force_wfmanager/ui/setup/mco/kpi_specification_view.py

```diff
@@ -100,44 +96,35 @@ def _model_views_default(self):
         """Creates a list of KPISpecificationModelViews for each
         model.kpi"""
         model_views = []
+
         if self.model is not None:
-            # Update model view list
-            model_views += [
-                KPISpecificationModelView(
-                    model=kpi,
-                    available_variables=self.kpi_name_options
-                )
-                for kpi in self.model.kpis
-            ]
+            # Add all MCO KPIs as ModelViews
+            for kpi in self.model.kpis:
+                model_view = KPISpecificationModelView(
+                        model=kpi,
+                        available_variables=self.kpi_name_options
+                    )
+                for var in self.kpi_name_options:
```

Comment:
During instantiation, if KPI already has a name, look for a matching `Variable` object to point to